lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -216,17 +225,51 @@ std::vector<SetGroupFAR>& SessionState::get_all_far_rules() {
   return FarList_;
 }
 
+SessionFsmState SessionState::get_state() {
+  return curr_state_;
+}
+
+magma::lte::Fsm_state_FsmState SessionState::get_smf_state()
+{
+  SessionFsmState  curr_state = get_state();
+  switch (curr_state) {
+    case CREATING:
+      return magma::lte::Fsm_state_FsmState_CREATING;
+	break;
+    case CREATED:
+      return magma::lte::Fsm_state_FsmState_CREATED;
+	break;
+    case ACTIVE:
+      return magma::lte::Fsm_state_FsmState_ACTIVE;
+	break;
+    case RELEASE:
+      return magma::lte::Fsm_state_FsmState_RELEASE;
+	break;
+    case INACTIVE:
+	 default:
+       return magma::lte::Fsm_state_FsmState_INACTIVE;
```

Comment:
The formatting looks weird here: `make format_session_manager`

Reply:
Fixed.